src/library/Avatar/Avatar.js

```diff
@@ -116,7 +88,7 @@ const Root = createStyledComponent(
       },
 
       '& > [role="img"]': {
-        fill: color
+        color
```

Comment:
why is `color` a better property for us to set than fill? it appears that setting `fill` overrides `color` so with this change fills can bleed in.

Reply:
Discussed offline. No change necessary.